src/controller/CHIPDeviceController.cpp

```diff
@@ -261,6 +261,14 @@ CHIP_ERROR DeviceController::LoadLocalCredentials(Transport::AdminPairingInfo *
 
         ReturnErrorOnFailure(admin->SetOperationalCert(ByteSpan(chipCert, chipCertLen)));
 
+        ChipLogProgress(Controller, "Getting ICA certificate for the controller from the issuer");
+        ReturnErrorOnFailure(mOperationalCredentialsDelegate->GetIntermediateCACertificate(0, cert, kMaxCHIPOpCertLength, certLen));
```

Comment:
It is not mandatory to have an ICAC in the chain. The path needs to account for optionality of the ICAC here. That is, there should be a way to determine absence from `GetIntermediateCACertificate` and skip the conversion and set the ICA to empty with `SetICACert`

Reply:
Will update.